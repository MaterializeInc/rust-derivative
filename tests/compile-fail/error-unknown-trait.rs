#[cfg(feature = "use_core")]
extern crate core;

#[macro_use]
extern crate derivative;

#[derive(Derivative)]
#[derivative(Baz)]
struct Foo;

#[derive(Derivative)]
#[derivative(PartialEq)]
struct Bar {
    #[derivative(Baz="wat")]
    bar: u8,
}

fn main() {}