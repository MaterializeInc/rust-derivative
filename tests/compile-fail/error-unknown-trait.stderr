error: unknown trait `Baz`
 --> $DIR/error-unknown-trait.rs:8:14
  |
8 | #[derivative(Baz)]
  |              ^^^

error: unknown trait `Baz`
  --> $DIR/error-unknown-trait.rs:14:18
   |
14 |     #[derivative(Baz="wat")]
   |                  ^^^
